lldb/test/API/python_api/type/TestTypeList.py

```diff
@@ -119,6 +119,11 @@ def test(self):
 
         self.assertEqual(task_type, task_head_pointee_type)
 
+        # Check whether we can find a nested type by name
+        name_type = task_type.FindNestedType("name")
```

Comment:
Should add tests for enums/unions as well?

Reply:
I can, but I'm not sure it would bring additional coverage.

Reply:
Up to you, though it would be nice to encode in the test-suite that we can in fact look up other tag-types with this API

Reply:
Done!